frontend/components/interests/form/InterestFormTextInput.tsx

```diff
@@ -49,6 +51,11 @@ const InterestFormTextInput = ({
     InputProps,
     minRows,
   };
+
+  if (disableAutocomplete) {
```

Comment:
Howcome the other instances of `InterestFormTextInput` do not have the `disableAutocomplete` and the autocomplete is off? Like the Description and Length of Campaign.

Reply:
Because your browser was unable to autocomplete description and length of campaign, so didn't bother suggesting any options for you.

Reply:
Okay